### ISSUE TITLE
Stopping ciliumginkgotests358_consul_1 ... error

### ISSUE THREAD

**Issue by tgraf** [cilium/cilium#2709]
https://jenkins.cilium.io/job/Cilium-Ginkgo-Tests/358/execution/node/30/log/

```
15:07:17 --- PASS: Test (0.00s)
15:07:17 PASS
15:07:17 OK: 1 passed
15:07:17 coverage: 72.1% of statements
15:07:17 ok  	github.com/cilium/cilium/pkg/workloads	0.096s
15:07:28 ?   	github.com/cilium/cilium/pkg/workloads/containerd	[no test files]
15:07:41 ?   	github.com/cilium/cilium/plugins/cilium-cni	[no test files]
15:07:55 ?   	github.com/cilium/cilium/plugins/cilium-docker	[no test files]
15:08:06 ?   	github.com/cilium/cilium/plugins/cilium-docker/driver	[no test files]
15:08:06 go tool cover -html=coverage-all.out -o=coverage-all.html
15:08:06 rm coverage-all.out
15:08:06 rm coverage.out
15:08:06 # Remove the networks
15:08:06 docker-compose -f test/docker-compose.yml -p $JOB_BASE_NAME-$BUILD_NUMBER down
15:08:06 Stopping ciliumginkgotests358_consul_1 ... 
15:08:06 Stopping ciliumginkgotests358_etcd_1 ... 
15:08:07 [2A[2K
Stopping ciliumginkgotests358_consul_1 ... error
[2B[1A[2K
Stopping ciliumginkgotests358_etcd_1 ... error
[1B
15:08:07 ERROR: for ciliumginkgotests358_etcd_1  cannot stop container: 37dd77c1685d4e7ff20e0d137eaaa64cd854e4ffe3b163f21a099b3b951fdef2: Cannot kill container 37dd77c1685d4e7ff20e0d137eaaa64cd854e4ffe3b163f21a099b3b951fdef2: unknown error after kill: docker-runc did not terminate sucessfully: container_linux.go:339: signaling init process caused "permission denied"
15:08:07 : unknown
15:08:07 
15:08:07 ERROR: for ciliumginkgotests358_consul_1  cannot stop container: 91813f1aa510f56789486855b3d44fa6c6db5ad89300453b1b2314e31fd4aa9c: Cannot kill container 91813f1aa510f56789486855b3d44fa6c6db5ad89300453b1b2314e31fd4aa9c: unknown error after kill: docker-runc did not terminate sucessfully: container_linux.go:339: signaling init process caused "permission denied"
15:08:07 : unknown
15:08:07 Removing network ciliumginkgotests358_default
15:08:07 network ciliumginkgotests358_default id 778d555941b6854e0ffc268a876e121eb6141e19997dd8f88f12b8e6afcc5d8e has active endpoints
15:08:07 Makefile:25: recipe for target 'tests-ginkgo' failed
15:08:07 make: *** [tests-ginkgo] Error 1
```

**Comment by eloycoto** [comment]
Attached full log. 
[consoleText.txt](https://github.com/cilium/cilium/files/1698226/consoleText.txt)

**Comment by tgraf** [comment]
I've submitted the respective change as individual PR:
https://github.com/cilium/cilium/pull/2730

**Comment by eloycoto** [comment]
Fixed  in the PR #2730
